src/pretix/control/forms/users.py

```diff
@@ -42,7 +42,7 @@ class UserEditForm(forms.ModelForm):
     error_messages = {
         'duplicate_identifier': _("There already is an account associated with this e-mail address. "
                                   "Please choose a different one."),
-        'pw_mismatch': _("Please enter the same password twice"),
+        'pw_mismatch': _("Please enter the same password twice")
```

Comment:
I think we can drop that change ;) The trailing comma is also generally considered a good idea in python, since it makes diffs easier to read. We currently do not enforce it, though.